Gems/Atom/RHI/DX12/Code/Source/RHI/TileAllocator.cpp

```diff
@@ -96,7 +96,7 @@ namespace AZ
                     if (!heap)
                     {
                         // Return directly if we can't create more heaps
-                        AZ_Error("TileAllocator", false, "Failed to create a heap page");
+                        AZ_Warning("TileAllocator", false, "Failed to create a heap page");
```

Comment:
Wouldnt the other warnings added above will trigger as well as this one? Also why switch from AZ_Error to AZ_Warning?

Reply:
Yes. Although, it's possible Heap allocator be used by any others (not at this moment). So I want to add the warning here so people would know the caller was from TileAllocator. 

It's warning but not an error because we have fall back solution (currently it's using default tile) later we may use mip eviction.